### PR TITLE
Adjust forum regex to only match forum posts

### LINKED ISSUE #61: The bot shouldn't tell me to post on the forum when I'm telling a user to post on the forum
https://github.com/AnalyticalGraphicsInc/cesium/issues/5742#issuecomment-322162603

### PR DESCRIPTION
**Pull request by ggetz**
Fixes #61

Only matches threads or message links to the `cesium-dev` forum.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
@ggetz, thanks for the pull request! Maintainers, we have a signed CLA from @ggetz, so you can review this at any time.

I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium-concierge/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).



__I am a bot who helps you make Cesium awesome!__ Thanks again.

**Comment by mramato** [comment]
Thanks, @ggetz! Taking a look now.

File: lib/commentOnClosedIssue.js [hunk 1]
```diff
@@ -85,4 +85,4 @@ commentOnClosedIssue._implementation = function (issueUrl, commentsUrl, reposito
         });
 };
 
-commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com[^\s.,:)\\]*/ig;
+commentOnClosedIssue._googleLinkRegex = /https?:\/\/groups\.google\.com\/forum\/(\?\w+=\w+)?#!(topic|msg)\/cesium-dev\/(\w+\/?)+/ig;
```
Comment (mramato) [inline lib/commentOnClosedIssue.js]: I think this is too strict now, for example these are all valid forum links that are not caught by this regex.

* https://groups.google.com/forum/?fromgroups#!topic/cesium-dev/bCzxAPBtX9k 
* https://groups.google.com/forum?fromgroups#!topic/cesium-dev/bCzxAPBtX9k 
* https://groups.google.com/d/msg/cesium-dev/bCzxAPBtX9k/EUGYilpgDgAJ



Perhaps we can simply check for `groups.google.com` `cesium-dev/` and make sure something is after `cesium-dev/`?

**Comment by ggetz** [comment]
@mramato Updated so it's less strict- I didn't see any of those variations when I searched the cesium repo the first time.

**Comment by mramato** [comment]
Looks great!  Ship it.

**Pull request merged**